assets/css/_diversion_page.scss

```diff
@@ -169,3 +169,24 @@
   height: 1.5rem;
   width: 1.5rem;
 }
+
+// A layout component which makes a `<dl/>` layout it's terms and definitions
+// on the same line
+.l-inline-dl {
+  // Display `<dt/>`'s and `<dd/>`'s on the same line
+  dt,
+  dd {
+    display: inline;
+  }
+
+  // But display each `<dt/>` on it's own line
+  dt::before {
+    content: "";
+    display: block;
+  }
+
+  // Remove leading margin from `<dd/>`'s
+  dd {
+    margin: 0;
+  }
+}
```

Comment:
I put this here because I don't think we have an intention to use this "component" anywhere else at this time, and it could be pulled out at that point, and it uses the `l-` prefix because this class _only_ pertains to layout of the list, and is not intended to do any visual styling related to fonts/colors/etc.